webrtc-stats.html

```diff
@@ -612,8 +612,48 @@ <h3>
           the first RTCP packet is sent or received that refers to the <a>SSRC</a> of the <a>RTP stream</a>.
         </p>
         <p>
-          RTP monitored objects are not [= deleted =].
+          RTP monitored objects are deleted when the RTP stream is reconfigured
+          with regards to the <a>SSRC</a> or {{RTCRtpStreamStats/codecId}}. If
+          the RTP stream is reconfigured such that it continues to be used, a
+          new RTP monitored object will be created with a new {{RTCStats/id}}.
         </p>
+        <p>
+          For a given RTP stats object its total counters must always increase,
+          but due to changes in SSRC or codec, an RTP stats object can be
+          deleted and replaced by another RTP stats object whose counters have
+          been reset. The caller should be aware of this when aggregating packet
+          counters accross multiple RTP stats objects.
+        </p>
+        <pre class="example highlight">
+          An RTCRtpSender is sending two layers of simulcast on SSRC=111 and
+          SSRC=222. Two "outbound-rtp" stats objects are observed (with
+          ssrc=111 and ssrc=222) and both objects' packet counters are
+          increasing.
+
+          One layer is inactivated using RTCRtpSender.setParameters(). While
+          this pauses one of the layers, the RTP monitored objects are not
+          deleted and the packet counter for the inactivated RTP stream reports
+          a non-increasing value.
+
+          Now the RTCRtpTransceiver is negotiated as "inactive". The RTP streams
```

Comment:
This is a hold case. Hold does not that you can forget about the SSRC, you must keep the objects.

Reply:
In your view, inactive should be treated the same as active=false regarding RTP lifetime? I.e. as long as the SSRC isn't changing, we don't delete?